dftimewolf/lib/containers/interface.py

```diff
@@ -17,23 +17,23 @@ class AttributeContainer():
   and private class members are not to be serialized.
 
   Attributes:
-    attributes: A list of generic attributes that can be used for passing
+    container_meta: A dict of container metadata that can be used for passing
```

Comment:
Can we just call this `meta` or `metadata`? `container_meta` sounds a bit redundant. (especially given that in the code we might call it `container.container_meta`)

Reply:
Done.